rir/src/interpreter/interp.h

```diff
@@ -5,15 +5,16 @@
 
 #undef length
 
-#include "call_support.h"
 #include "interp_context.h"
 #include "interp_data.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-SEXP evalRirCode(Code* c, Context* ctx, rir::EnvironmentProxy* ep);
+struct DispatchContext;
+SEXP evalRirCode(Code* c, Context* ctx, SEXP* env,
+                 const DispatchContext* = nullptr);
```

Comment:
I would maybe prefer being explicit with the nullptr dispatch context, since one needs to look at the declaration to see it